src/kem/kyber/types.rs

```diff
@@ -1,7 +1,7 @@
 macro_rules! impl_generic_struct {
     ($name:ident) => {
         pub struct $name<const SIZE: usize> {
-            value: [u8; SIZE],
+            pub(crate) value: [u8; SIZE],
```

Comment:
Let's not make it so visible ...

```suggestion
            pub(in super) value: [u8; SIZE],
```